vignettes/rvar.Rmd

```diff
@@ -96,9 +96,10 @@ post
 ```
 
 The `draws_rvars()` datatype works much the same way that other `draws` formats
-do; see **TODO: INSERT LINK** for an introduction to `draws` objects. One difference is that
-`draws_rvars` counts variables differently, because it allows variables to be multidimensional.
-For example, the `post` object above contains two variables, `mu` and `Sigma`:
+do; see the main [posterior vignette](posterior.html) for an introduction to
```

Comment:
I don't know all the pros/cons; is this vignette inter-linking format preferable to `vignette("posterior")`, which will be auto-linked by {pkgdown}?

Reply:
I didn't realize pkgdown would auto link it. We should just use `vignette("posterior")` in that case.

Reply:
Will change it in both vignettes